modules/exploits/multi/misc/weblogic_deserialize_badattrval.rb

```diff
@@ -0,0 +1,771 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Exploit::Remote::Tcp
+  include Exploit::CmdStager
+  include Exploit::Powershell
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Weblogic Deserialize BadAttributeValueExpException',
+      'Description'    => %q(
+        There exists an object deserialization vulnerability
+        in multiple versions of WebLogic.
+
+        Unauthenticated remote code execution can be achieved
+        by sending a serialized BadAttributeValueExpException object
+        over T3 to vulnerable WebLogic servers.
```

Comment:
```suggestion
        There exists a Java object deserialization vulnerability
        in multiple versions of WebLogic.

        Unauthenticated remote code execution can be achieved
        by sending a serialized BadAttributeValueExpException object
        over the T3 protocol to vulnerable WebLogic servers.
```

And https://github.com/rapid7/metasploit-framework/pull/13463/files#r427423481.